src/SFA.DAS.AssessorService.Domain/Consts/CompletionStatus.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace SFA.DAS.AssessorService.Domain.Consts
+{
+    public enum CompletionStatus
+    {
+        Continuing = 1,
+        Complete = 2,
+        Withdrawn = 3,
```

Comment:
Are these values being used as 'Magic numbers' elsewhere - could you correct that?